latch_cli/workflow_config.py

```diff
@@ -0,0 +1,33 @@
+import json
+from dataclasses import asdict, dataclass
+from datetime import datetime
+from pathlib import Path
+
+from pkg_resources import get_distribution
+
+from latch_cli.constants import latch_constants
+
+
+@dataclass(frozen=True)
+class LatchWorkflowConfig:
+    """Configuration for a Latch workflow."""
+
+    latch_version: str
+    """Version of the Latch SDK used to initialize the workflow"""
+    base_image: str
+    """Exact version of the included workflow base image"""
+    date: str
+    """Timestamp of the `latch init` call"""
+
+
+def create_and_write_config(base_image: str, pkg_root: Path):
```

Comment:
see my comment above about restructuring the signature here

Reply:
also adding on, `base_image` should be a member of the enum you had above, and you should construct the full docker url here instead of passing it in